Wallet unlock form improvements

keerifox opened the pull request:
password input and unlock button are now disabled for 0.5s upon unsuccessful unlock attempt, rather than delaying any unlocking attempt by 0.5s (even if the password was entered correctly from the first try, the unlocking process used to be delayed)

![image](https://user-images.githubusercontent.com/29272208/104504456-61625d80-55da-11eb-8ebb-26dd59b50437.png)


also changes wording from "Invalid password" to "Incorrect password"

keerifox committed:
500ms wallet unlock delay is now only present if attempted unlock within last 2s

keerifox committed:
fix lint

Joohansson commented on src/app/components/wallet-widget/wallet-widget.component.ts:
```diff
@@ -67,7 +69,24 @@ export class WalletWidgetComponent implements OnInit {
   async unlockWallet() {
-    await new Promise(resolve => setTimeout(resolve, 500)); // brute force delay
+    if (this.shouldDelayUnlockAttempt === true) {
+      await new Promise(resolve => setTimeout(resolve, 500)); // brute force delay
```
I think some additional logic must be set here. Since the function is async you can press the button and bruteforce how much you want. The delay has no affect? Maybe set a global variable that is reset after 500ms. And if that variable is set, just return the unlockWallet function. I made this wrong from the very beginning it seems.

keerifox committed:
wallet unlock form improvements

password input and unlock button are now disabled for 0.5s upon unsuccessful unlock attempt
change wording from "Invalid password" to "Incorrect password"

keerifox committed:
fix lint
